### PR TITLE
Otherwordly patron: the toon

### PR DESCRIPTION
**Pull request by sammuns**
Warlock subclass from the middle finger of vecna.

### CODE REVIEW

File: subclass/thetoonwarlockpatron.json [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                "targetSchema": "1.0.8"
+            }
+        ],
+        "dateAdded": 1549715846,
+        "dateLastModified": 1660453920
+    },
+    "subclass": [
+        {
+            "name": "The Toon",
+            "source": "WarlockToon",
```
Comment (TheGiddyLimit) [inline subclass/thetoonwarlockpatron.json:26]:
```suggestion
            "source": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                            "passwall"
+                        ]
+                    }
+                }
+            ],
+            "subclassFeatures": [
+                "The Toon|Warlock||The Toon|WarlockToon|1",
+                "Slaspstick|Warlock||The Toon|WarlockToon|6",
+                "Two-Dimensional|Warlock||The Toon|WarlockToon|10",
+                "Anvil on Head|Warlock||The Toon|WarlockToon|14"
```
Comment (TheGiddyLimit) [inline subclass/thetoonwarlockpatron.json:72]:
```suggestion
                "The Toon|Warlock||The Toon|MFOV:TT|1",
                "Slaspstick|Warlock||The Toon|MFOV:TT|6",
                "Two-Dimensional|Warlock||The Toon|MFOV:TT|10",
                "Anvil on Head|Warlock||The Toon|MFOV:TT|14"

```

File: subclass/thetoonwarlockpatron.json [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+            ]
+        }
+    ],
+    "subclassFeature": [
+        {
+            "name": "The Toon",
+            "source": "WarlockToon",
```
Comment (TheGiddyLimit) [inline subclass/thetoonwarlockpatron.json:79]: etc. etc.

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
Sources need fixing

File: subclass/thetoonwarlockpatron.json [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+                "Two-Dimensional|Warlock||The Toon|MFOV:TT|10",
+                "Anvil on Head|Warlock||The Toon|MFOV:TT|14"
+
+            ]
+        }
+    ],
+    "subclassFeature": [
+        {
+            "name": "The Toon",
+            "source": "WarlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:80]: You need to change all the `WarlockToon` source references, as it does not exist. The only source in the meta is `MFOV:TT`:

```suggestion
            "source": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+            "className": "Warlock",
+            "classSource": "PHB",
+            "subclassSource": "warlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:83]: Same thing:

```suggestion
            "subclassSource": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+            "subclassShortName": "The Toon",
+            "page": 1,
+            "level": 1,
+            "entries": [
+                "Out on the edge of the multiverse is a demiplane called Toontown, a wacky place inhabited by toons, creatures of ink and magic that defy logic. In Toontown, the laws of space and time take a wrong turn and dive straight off a cliff; it’s an eternity of loony hijinks, a torrent of silliness, a plane that despises seriousness. Toons take many forms -- often little critters or animate objects with exaggerated eyes drawn on them -- but the Barons of Toontown are known far and wide. These powerful (and copywritten) toons can let you bend the laws of space and time, provided that you leave a trail of slapstick jokes and utter silliness in your wake.",
+                {
+                    "type": "refSubclassFeature",
+                    "subclassFeature": "Expanded Spell List|Warlock||The Toon|WarlockToon|1|WarlockToon"
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:91]:
```suggestion
                    "subclassFeature": "Expanded Spell List|Warlock||The Toon|MFOV:TT|1|MFOV:TT"
```

File: subclass/thetoonwarlockpatron.json [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+                },
+                {
+                    "type": "refSubclassFeature",
+                    "subclassFeature": "Big Pockets|Warlock||The Toon|WarlockToon|1|WarlockToon"
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:95]:
```suggestion
                    "subclassFeature": "Big Pockets|Warlock||The Toon|MFOV:TT|1|MFOV:TT"
```

File: subclass/thetoonwarlockpatron.json [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+                },
+                {
+                    "type": "refSubclassFeature",
+                    "subclassFeature": "Zany Limbs|Warlock||The Toon|WarlockToon|1|WarlockToon"
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:99]:
```suggestion
                    "subclassFeature": "Zany Limbs|Warlock||The Toon|MFOV:TT|1|MFOV:TT"
```

File: subclass/thetoonwarlockpatron.json [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+                }
+            ]
+        },
+        {
+            "name": "Expanded Spell List",
+            "source": "WarlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:105]:
```suggestion
            "source": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 10]
```diff
@@ -0,0 +1,221 @@
...
+            "className": "Warlock",
+            "classSource": "PHB",
+            "subclassSource": "warlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:108]:
```suggestion
            "subclassSource": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+                            "5th",
+                            "{@spell mislead}, {@spell passwall}"
+                        ]
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Big Pockets",
+            "source": "WarlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:152]:
```suggestion
            "source": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 12]
```diff
@@ -0,0 +1,221 @@
...
+            "page": 1,
+            "level": 1,
+            "header": 2,
+            "entries": [
+                "Starting at 1st level, you scoff at lengthy encumbrance rules. When you open a pocket, bag, or other container that you touch, you can choose for it to lead to your personal extradimensional space, which is 64 cubic feet in volume. The container’s opening stretches to accommodate items of any size which can fit within the space, and items within the space are weightless until removed. When you reach into this space, any item you intend to take is magically on top. A container loses this property when you are no longer touching it."
+            ]
+        },
+        {
+            "name": "Zany Limbs",
+            "source": "WarlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:166]:
```suggestion
            "source": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 13]
```diff
@@ -0,0 +1,221 @@
...
+            "page": 1,
+            "level": 1,
+            "header": 2,
+            "entries": [
+                "At 1st level, you have elastic, fantastic limbs. You can stretch your arms and legs up to 5 feet beyond their normal length, allowing you to interact with distant objects or creatures, and your melee weapon attacks have the Reach property. Additionally, you can ignore the Heavy property of weapons."
+            ]
+        },
+        {
+            "name": "Slaspstick",
+            "source": "WarlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:180]:
```suggestion
            "source": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 14]
```diff
@@ -0,0 +1,221 @@
...
+            "page": 1,
+            "level": 6,
+            "header": 2,
+            "entries": [
+                "By at 6th level, you have a knack for cartoon violence. When you take bludgeoning, piercing, or slashing damage while you are standing, you can use your reaction to halve the damage taken and also be knocked prone, as you are either squashed flat, deflated like a balloon, or momentarily chopped in half."
+            ]
+        },
+        {
+            "name": "Two-Dimensional",
+            "source": "WarlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:194]:
```suggestion
            "source": "MFOV:TT",
```

File: subclass/thetoonwarlockpatron.json [hunk 15]
```diff
@@ -0,0 +1,221 @@
...
+            "page": 1,
+            "level": 10,
+            "header": 2,
+            "entries": [
+                "At 10th level, when comically convenient, you can use your bonus action to turn sideways and become completely flat or to return to return to three-dimensions. While two-dimensional, you can fit through passages as narrow as 1/2 inch across, and you have half cover from ranged attacks. Additionally, you can also glide on the wind when you fall, floating 2 feet horizontally for each foot downward you fall. However, you are far more fragile, and have vulnerability to bludgeoning, piercing, and slashing damage."
+            ]
+        },
+        {
+            "name": "Anvil on Head",
+            "source": "WarlockToon",
```
Comment (Git-GoR) [inline subclass/thetoonwarlockpatron.json:208]:
```suggestion
            "source": "MFOV:TT",
```

**Review by Git-GoR — CHANGES_REQUESTED** [review]
There are still references to the wrong source that need to be changed.

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
Please see: https://github.com/TheGiddyLimit/homebrew/pull/1430/checks

Also, this file should be called e.g. `Mage Hand Press; The Toon Warlock Patron.json`

**Comment by TheGiddyLimit** [comment]
superseded by #2154

**Pull request closed without merging**